src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -77,10 +76,7 @@ class AutoRerouteSpec extends HealthCheckSpecification {
 
         cleanup: "Revive the ISL back (bring switch port up) and delete the flow"
         flow && flowHelperV2.deleteFlow(flow.flowId)
-        islToFail && antiflap.portUp(islToFail.srcSwitch.dpId, islToFail.srcPort)
-        wait(discoveryInterval + WAIT_OFFSET) {
-            assert northbound.getActiveLinks().size() == topology.islsForActiveSwitches.size() * 2
-        }
+        islHelper.restoreIsl(islToFail)
```

Comment:
islToFail && islHelper.restoreIsl(islToFail) and the same for ISL restoring in the cleanup sections below.

Reply:
This is exhaustive because `restoreIsl()` method is failsafe (prepared to automatic cleanup)